### DISCUSSION TITLE
Off- and Online cracking time

### DISCUSSION THREAD

**Discussion by bakancs** [StellarSand/IYPS#12] [Ideas]
There should be calculated online and offline password cracking time.

**Comment by StellarSand** [comment]
Actually there already is, it's just displayed differently. 
- 10B guesses/second and 10k guesses/second are offline ones
- 10 guesses/second and 100 guesses/hour are online ones

It's done this way because it makes more sense to display the actual guesses per second instead of just showing "online" or "offline". 
Both online and offline methods have a number of variables affecting them like  number of CPUs or CPU cores available, frequency of the CPU cores, CAPTCHAs, network latency, throttling etc. etc., which ultimately affects how many actual guesses per second is being used to crack the passwords, so that's what is being displayed.
